src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,75 @@
-const Countdown = () => null;
+import { useState, useEffect } from "react";
 
-export default Countdown;
+
+const StopWatch = () => {
```

Comment:
stopwatch time is not customizable - should be passed in as a prop, or somehow configurable.